app/assets/javascripts/components/QuotesBox.js.jsx

```diff
@@ -108,6 +108,9 @@ class QuotesBox extends React.Component{
             <div className="btn btn-success grey darken-1 shadows fix-that-width" onClick={this.nextQuote}>Next</div>
             <div className="row">
               {favorite_icon}
+              <a href={`https://twitter.com/intent/tweet?text=${quote}`} target="blank">
```

Comment:
I used a [template literal](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals). You were using arrow functions so I assumed es6 was okay.